README.md

```diff
@@ -287,8 +287,10 @@ i  starting release process (may take several minutes)...
 
 ## Deploying to the official site
 
-Usually, official site deploys are performed by Travis. In the event that you
-need to manually deploy, use the deploy script and the `default` project:
+As of Feb 23, 2021, official site deploys are
+performed by GitHubActions (GHA).
```

Comment:
There should be a space between "GitHub" and "Actions".

Maybe use GHA everywhere else instead? (Or drop the "(GHA)" everywhere.)